### PR TITLE
Game prototype

### PR DESCRIPTION
**Pull request by marcbue**
This is the prototype of the game to get a first idea about separation in MVC-Models and have a first directory structure.

### CODE REVIEW

File: src/model/model.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        return self.name
+
+    def set_color(self, color):
+        self.color = color
+
+    def get_color(self):
+        return self.color
+
+    def store_data(self):
+        pickle.dump(self, open("save.p", "wb"))
```
Comment (DorotheaMueller) [inline src/model/model.py]: Please save as [player name].p.

Comment (DorotheaMueller) [inline src/model/model.py]: Please specify which file should be opened.

File: src/controller/controller.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+from model.model import Player
+from view.view import View
+
+class Controller:
+    def __init__(self):
+        self.player = Player()
+        self.view = View()
+        self.run()
+    
+    def run(self):
```
Comment (DorotheaMueller) [inline src/controller/controller.py]: Unit test?

**Comment by DorotheaMueller** [comment]
For getters and setters, we do not necessarily need comments, but please comment the rest of the code :)

File: src/model/model.py (hunk 1 above)
Comment (marcbue) [reply src/model/model.py]: done

Comment (marcbue) [reply src/model/model.py]: done

File: src/view/view.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        self.view = "start_screen"
+        size = width, height = 329, 249
+        self.screen = pygame.display.set_mode(size)
+
+    def get_event(self):
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                return "quit"
+            else:
+                return False
```
[the remark this answers is not in the document]
Comment (marcbue) [reply src/view/view.py]: commented

File: test/test_model.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+def test_name_getter():
+    player = Player(name, color)
+    assert player.get_name() == name
+
+
+def test_color_setter():
+    player = Player(name, color)
+    new_color = "herbert"
```
[the remark this answers is not in the document]
Comment (marcbue) [reply test/test_model.py]: done

File: test/test_model.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    player.set_color(new_color)
+    assert player.color == new_color
+
+
+def test_color_getter():
+    player = Player(name, color)
+    assert player.get_color() == color
+
+
+def test_data_storage():
```
[the remark this answers is not in the document]
Comment (marcbue) [reply test/test_model.py]: done

**Pull request merged**